### DISCUSSION TITLE
three questions on Related publications

### DISCUSSION THREAD

**Discussion by shiranD** [alshedivat/al-folio#2054] [Q&A] [answered]
**Acknowledge the following**
![question_2](https://github.com/alshedivat/al-folio/assets/9203906/4b37c649-6ad5-4a76-9be9-32ad6b435014)
![question_3](https://github.com/alshedivat/al-folio/assets/9203906/c61838ee-1667-41bf-beeb-4407299ad5d7)
![question_3_config](https://github.com/alshedivat/al-folio/assets/9203906/d2e3f3c2-d7d0-41b6-8aae-cf70bbd609ea)

- [y] I carefully read and followed the [Getting Started](https://github.com/alshedivat/al-folio#getting-started) guide.
- [y] I read through [FAQ](https://github.com/alshedivat/al-folio#faq) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my issue.
- [y] The issue I am raising is a potential bug in al-folio and not just a usage question. <br> [For usage questions, please post in the [Discussions](https://github.com/alshedivat/al-folio/discussions) instead of raising an issue.]

**Screenshots**

**System (please complete the following information):**
 - OS: iOS ventura 13.6.1. m2
 - Browser (and its version) safari
 - Jekyll version I am not sure, would be glad for guidance. 
- Ruby version not sure, would be glad for guidance.
- Both were installed on the Docker image created by the recipe in the repo.

**Additional context**
The solution described by both having a related link as well as pointing to it through {% reference ahner_self-healing_2015 %} for instance worked well for me but I still have questions as described [here](https://github.com/alshedivat/al-folio/issues/1193). 
1) What if I'd like to refer to a paper I did not take part of (or does not appear on my papers.bib).

2) How can the citation style work similar to https://buerviper.github.io/projects/self-healing repo? Mine shows a long string concatenating the fields of bib. The first image is relevant to the question.

3) even though I have modified in _confing.yaml scholar: first and scholar: last names written in square brackets it still does not highlight them as suggested. Two last images are relevant to the question.

Happy for any solution to any of these questions. And happy to provide further details on the system. Shiran

**Comment by george-gca** [comment]
1. Currently this is not supported, but I think the best way to implement something like this is adding the paper you did not take part in the bib file and then adding a new key, something like `not_mine=false`, then in the publications tab only display the papers without `not_mine` or with `not_mine=false`.
2. Since we use the [jekyll-scholar](https://github.com/inukshuk/jekyll-scholar) plugin to handle this, check its documentation. Also check if the site you gave as an example is not open sourced. If it is, you can simply check its code.
3. I believe you inverted the first name and last names there.

**Comment by george-gca** [comment, marked as answer]
Just updated the example blog post with citation examples in #2056. You can change the citation style in [_config.yml](https://github.com/alshedivat/al-folio/blob/e3997d0f6cff865270b3abf645c289f2c3f519c5/_config.yml#L281) to any of the ones supported in [jekyll-scholar](https://github.com/inukshuk/jekyll-scholar).

Actually, for the first question check [jekyll-scholar](https://github.com/inukshuk/jekyll-scholar?tab=readme-ov-file#citations-when-theres-more-than-one-bibliography) documentation. You can have 2 bibliography files, and cite papers from both.

**Comment by shiranD** [comment]
This helped me greatly. Thanks for catching my (embarrassing) mistake on the 3rd question. My name is underlined and not colored but it's definitely better.
